### PR TITLE
Apply filter in the file uploads

### PR DESCRIPTION
**Pull request by josehernandezfintecheandomx**
## Description

Set in some views that upload file(s) the filter according to the file that can be uploaded, like image or just spreadsheet file

## Related issues and discussion
#{Issue Number}

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] If you have multiple commits please combine them into one commit by squashing them.

- [ ] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

File: src/app/clients/clients-view/custom-dialogs/upload-image-dialog/upload-image-dialog.component.html [hunk 1]
```diff
@@ -2,11 +2,11 @@ <h1 mat-dialog-title>Upload Client Image</h1>
 
 <div>
 
-  <mifosx-file-upload flex="60%" (change)="onFileSelect($event)"></mifosx-file-upload>
+  <mifosx-file-upload flex="60%" (change)="onFileSelect($event)" acceptFilter=".jpg,.jpeg,.gif,.tif,.tiff,.png|image/*"></mifosx-file-upload>
```
Comment (adamsaghy) [inline src/app/clients/clients-view/custom-dialogs/upload-image-dialog/upload-image-dialog.component.html:5]: Looks incorrect to me.
Based on the default filter on the backend the following are whitelisted: .jpg,.jpeg,.png.

File: src/app/clients/clients-view/custom-dialogs/upload-signature-dialog/upload-signature-dialog.component.html [hunk 2]
```diff
@@ -2,11 +2,11 @@ <h1 mat-dialog-title>Upload Client Signature</h1>
 
 <div>
 
-  <mifosx-file-upload flex="60%" (change)="onFileSelect($event)"></mifosx-file-upload>
+  <mifosx-file-upload flex="60%" (change)="onFileSelect($event)" acceptFilter=".jpg,.jpeg,.gif,.tif,.tiff,.png|image/*"></mifosx-file-upload>
```
Comment (adamsaghy) [inline src/app/clients/clients-view/custom-dialogs/upload-signature-dialog/upload-signature-dialog.component.html:5]: Looks incorrect to me.
Based on the default filter on the backend the following are whitelisted: .jpg,.jpeg,.png.

Comment (josehernandezfintecheandomx) [reply src/app/clients/clients-view/custom-dialogs/upload-signature-dialog/upload-signature-dialog.component.html]: Done

File: src/app/shared/file-upload/file-upload.component.ts [hunk 3]
```diff
@@ -9,15 +9,23 @@ import { Component, Input, Output, EventEmitter } from '@angular/core';
...
+  @Input() acceptFilter: string;
+
   /** Selected file name */
   fileName: File;
 
   constructor() { }
 
+  ngOnInit() {
+    if (!this.acceptFilter) {
+      this.acceptFilter = '.xls,.xlsx,.pdf,.doc,.docx,.txt,.png,.jpeg,.jpg';
```
Comment (adamsaghy) [inline src/app/shared/file-upload/file-upload.component.ts:25]: I dont think "txt" is by default enabled on the backend

Comment (josehernandezfintecheandomx) [reply src/app/shared/file-upload/file-upload.component.ts]: Done! removed the txt file extension

**Review by adamsaghy — APPROVED** [review]
LGTM

**Pull request merged**